201-decrypt-vmss-windows/azuredeploy.json

```diff
@@ -2,13 +2,13 @@
   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "vmssName": {
+    "existingVmssName": {
```

Comment:
since this has an existing resource requirement, you need to provide a prereq template to provision those resources - see the readme/contribution guide